bindings_generator/src/classes.rs

```diff
@@ -100,6 +114,18 @@ pub enum {class_name}{enum_name} {{"#,
             continue;
         }
 
+        // Check if the value is a duplicate. This is fine because `values` is already sorted by value.
```

Comment:
Nice! This does mean that the APIs might look different in Rust than in regular Godot. Since `values` is sorted by value (and I assume it is a stable sort) only the first enum variant will ever be generated. That should be fine in general, but things like `RPC_MODE_SLAVE` would be kept as they would be generated first. Also with the `CameraServer` enum it seems a bit weird to skip them, as they do express different things.

But I am not sure what to do about this, I think this is fine for now, but maybe we can find a way to allow aliases later on.

Reply:
Variants manually skipped are checked earlier than this, so at least things like `RPC_MODE_SLAVE` will be fine. Future ones can be weird, though, but at least it'll compile.